### DISCUSSION TITLE
task view bug

### DISCUSSION THREAD

**Discussion by 3weditos** [valinet/ExplorerPatcher#1882] [Q&A]
Hello
im using explorer patcher since 1 month
there is a problem just happen to me :
alt + tab and other tabbing shortcuts not working if i press win(button) + tab to open the task view button
and i have to restart explorer to make alt + tab and the others work again 
but really i need to fix this problem because i always use the task view button 
please help
win 11 ver : 22H2 (OS Build 22621.2134)
explorer patcher ver : latest

**Comment by Poopooracoocoo** [comment]
Closing as duplicate of https://github.com/valinet/ExplorerPatcher/issues/1687 and https://github.com/valinet/ExplorerPatcher/issues/1802
